website/docs/reading/bazel.md

```diff
@@ -0,0 +1,91 @@
+---
+id: bazel
+title: Prysm's Build System, Bazel
+sidebar_label: Why Bazel
+---
+
+This page explains why Prysm uses a special build system, called [Bazel](https://bazel.build) to compile everything in our mono-repository.
+
+## Why Bazel?
+
+Instead of using the `Go` tool to build Prysm, our team relies on the [Bazel](https://bazel.build) build system used by major companies such as Google, Uber, Coinbase, and more to manage monorepositories. Bazel provides reproducible builds and a sandboxed environment that ensures everyone building Prysm has the same experience and can build our entire project from a single command. The rationale for why we choose this system comes down to three major problems we need solved as part of our software development process:
```

Comment:
```suggestion
Instead of using the `Go` tool to build Prysm, our team relies on the [Bazel](https://bazel.build) build system used by major companies such as Google, Uber, Coinbase, and more to manage monorepositories. Bazel provides reproducible builds and a sandboxed environment that ensures everyone building Prysm has the same experience and can build our entire project from a single command. The rationale for why we chose this system comes down to three major problems we need solved as part of our software development process:
```